keyvault/client/src/main/java/com/azure/keyvault/implementation/Page.java

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.keyvault.implementation;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.List;
+
+/**
+ * An instance of this class defines a page of Azure App Configuration resources and a link to get the next page of
+ * resources, if any.
+ *
+ * @param <T> type of Azure App Configuration resource to deserialize.
+ */
+public final class Page<T> {
```

Comment:
@conniey Did you ever push this implementation API into azure-common? Is that still the plan?

Reply:
There was resistance to this change when I created it originally. I can recreate this PR in our main repo.

Reply:
I think you should - I don't want to have this class duplicated across all client libraries. Thanks.

Reply:
Noted, needs to be moved to Azure common.

Reply:
It's been merged in now. :) I added you on the PR that my branch uses it.